pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -229,11 +229,11 @@ spec:
           value: "$(context.pipelineRun.uid)/data.json"
         - name: requester
           value: $(tasks.extract-requester-from-release.results.output-result)
+        - name: commonTag
```

Comment:
This doesn't *really* change anything in the pipeline, but can you switch the order in the yaml of `push-snapshot` and `rh-sign-image`? Now that `rh-sign-image` has a parameter that is a result of `push-snapshot`, it will always run after it (of course, since you are removing the `runAfter` you know this). But it is a little confusing to me to have a task that runs first show up second in the pipeline definition

Reply:
I can move the actual definition in the yaml if it is more intuitive.

Reply:
I think it would be, yes

Reply:
moved